packages/material-ui/src/utils/getScrollbarSize.js

```diff
@@ -0,0 +1,17 @@
+// A change of the browser zoom change the scrollbar size.
+function getScrollbarSize() {
```

Comment:
What is the difference to `dom-helpers`?

Reply:
I have removed the isDOM dependency and the recalc flag.

Reply:
I guess we need to add a link to the original source for credit. Honestly I don't really see a point of dropping it. It's just a slightly stripped down version that adds maintenance burden for us. That's not worth saving 20B or something like that. This comes down to (again) reducing the apparent bundle size that will actually result in a bigger application bundle for anybody that already uses `dom-helpers` scrollBarSize.

Reply:
- There [is close to zero maintenance](https://github.com/react-bootstrap/dom-helpers/graphs/contributors) burden for us. There is close to zero activity on the project.

![Capture d’écran 2019-03-14 à 11 52 19](https://user-images.githubusercontent.com/3165635/54351376-a4a2bc80-464f-11e9-8278-30541e8d9665.png)

- This dependency doesn't align with our goal. We have been progressively moving away from it. We don't need old browser support.

![Capture d’écran 2019-03-14 à 11 48 25](https://user-images.githubusercontent.com/3165635/54351127-17f7fe80-464f-11e9-8421-96eaed92acad.png)

- Sure, or we could credit: 
https://github.com/twbs/bootstrap/blob/3ffe3a5d82f6f561b82ff78d82b32a7d14aed558/js/src/modal.js#L512-L519. I have looked at the git history, Bootstrap has historical precedence.
- The gain isn't apparent. react transition group [uses](https://github.com/reactjs/react-transition-group/blob/c50e1dd5d413822e4609bffde2a7ab3ef074b1d3/package.json#L57
) this dependency, and yet, the reported size decreases.

Reply:
Makes sense. I'm just scared of having any code that depends on browser vendor implementations. But we live in better times now so let's move forward.